lib/open-amp/README

```diff
@@ -0,0 +1,21 @@
+Description:
```

Comment:
README file is not standard in zephyr( zephyrbots report), so I will move the description to resource_table.c as a file header comment.